luarules/gadgets/game_prevent_excessive_share.lua

```diff
@@ -5,7 +5,7 @@ function gadget:GetInfo()
 		author  = 'Niobium',
 		date    = 'April 2012',
 		license = 'GNU GPL, v2 or later',
-		layer   = 0,
+		layer   = 2,
```

Comment:
```suggestion
		layer   = 2, -- after 'Tax Resource Sharing'
```

Reply:
Oops. I was thinking to put it to 1/2 because tax_sharing rejects the original request, so to let any future gadgets using AllowResourceTransfer and performing side effects to actually run, when sharing_tax modoption is being used.